source/connect/connection-targets.txt

```diff
@@ -0,0 +1,120 @@
+.. _kotlin-sync-connection-targets:
+
+==========================
+Choose a Connection Target
+==========================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: connection string, URI, server, settings, client
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use a connection string and a ``MongoClient`` object
+to connect to different types of MongoDB deployments.
+
+Atlas
+-----
+
+To connect to a MongoDB deployment on Atlas, include the following elements
+in your connection string:
+
+- The URL of your Atlas cluster
+- Your MongoDB username
+- Your MongoDB password
+
+Then, pass your connection string to the ``MongoClient`` constructor.
+
+.. tip::
+
```

Comment:
nit: tck param looks wrong (admittedly I dont know what it actually does 😁)

Reply:
Apologies: `Atlas driver connection guide </driver-connection?tck=docs_driver_python>`
Specifically: `docs_driver_python`

Reply:
Ah, yes, I can remove that. Good eye!